server/bleep/src/semantic.rs

```diff
@@ -332,6 +332,18 @@ fn make_kv_filter(key: &str, value: &str) -> FieldCondition {
     }
 }
 
+fn make_kv_filter_strict(key: &str, value: &str) -> FieldCondition {
```

Comment:
Should this also be used in the `delete_points_by_path` function?

Reply:
yes it should. made this a separate utility in order to avoid breaking existing behaviour. will swap it out and give this a test.

Reply:
works as expected from a quick test:
- index a repo
- count points in qdrant
- delete tantivy index
- re-index the repo
- count points in qdrant again

the count should not double, it should stay the same.